src/main/kotlin/org/codefreak/codefreak/service/file/IFileCollectionRepository.kt

```diff
@@ -0,0 +1,10 @@
+package org.codefreak.codefreak.service.file
+
+import java.util.UUID
+
+interface IFileCollectionRepository {
+  fun saveChanges(collectionID: UUID, commitMessage: String, force: Boolean): Boolean
+  fun resetAndLoadVersion(collectionID: UUID, versionID: Int, commitMessage: String): Boolean
+  fun getCommitMessages(collectionID: UUID): MutableMap<Int, String>
+  fun getCommitMessage(collectionID: UUID, commitId: String): String
+}
```

Comment:
Please document what the interface and it's methods do. I'm other classes we will use the interface and not the implementing class, so we need some documentation here.